### ISSUE TITLE
Adding physics material to map

### ISSUE THREAD

**Issue by EvilWitch** [Seanba/SuperTiled2Unity#113]
Hello,
I've been having an issue that I haven't found clear answer anywhere. I want to add physics 2D material to Rigidbody 2d of the map collison, however, every time I run the project it reverts to None. If I add physics material when the project is running, it works perfectly fine. Do I need do change something in settings, or I need to look somewhere else?
![Przechwytywanie](https://user-images.githubusercontent.com/56895324/69834304-d4274280-1239-11ea-9bcb-715b0f26a4ca.JPG)
Unity version I'm using is 2019.2.10f1

**Comment by Seanba** [comment]
Hi there, @EvilWitch. It looks like you've discovered an issue with the prefab overrides in scripted importers. I think I have a work-around that has an unfortunate side-effect (our `*.unity` scene files will store extra/bloated in them). I'll play around with this some more and enter a bug with Unity if need be. Thanks.

**Comment by Seanba** [comment]
Version 1.9.0 of SuperTiled2Unity fixes the prefab instance overrides problem you've run into here.

[https://seanba.itch.io/supertiled2unity/devlog/112310/supertiled2unity-version-190](url)

IMO this is a significant improvement for ST2U and I probably would have remained unaware of the issue for some time without your bug report. Thanks for letting me know and good luck with your game.

**Comment by TimCreasman** [comment]
@Seanba I figured this was the best place to ask this considering EvilWitch's question is very similar to an issue I am having and seems to be directly related to their question. If you wish for me to open a **new issue** I can do that as well. 

Anyways, is there a way to set a default Physics Material to the collision layer from within Tiled itself? I have a specific physics material I wish to apply across the whole Collision_Default layer. I noticed the option for Material Matching but this seems to be related to rendering materials and not physics materials. 

Thank you for all the hard work you've put into this project!
